middle_end/flambda2/terms/placement.ml

```diff
@@ -0,0 +1,34 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                      Vincent Laviron, OCamlPro                         *)
+(*                                                                        *)
+(*   Copyright 2022 OCamlPro SAS                                          *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+type t =
+  | Delay
+  | Strict
+
+let [@ocamlformat "disable"] print ppf dup =
+  match dup with
+  | Delay -> Format.fprintf ppf "duplicatable"
+  | Strict -> Format.fprintf ppf "non-duplicatable"
```

Comment:
The printing should match the constructors

Reply:
(also no need for a "disable" attribute)

Reply:
fixed